Cargo.toml

```diff
@@ -68,13 +60,13 @@ default = [
 ]
 runtime = [
     "tcp",
-    "tokio",
+    #"tokio",
```

Comment:
no longer optional?

Reply:
It should be, adding back in the features.